src/isomorphic/classic/types/ReactPropTypes.js

```diff
@@ -110,10 +111,21 @@ function createChainableTypeChecker(validate) {
     propName,
     componentName,
     location,
-    propFullName
+    propFullName,
+    secret
   ) {
     componentName = componentName || ANONYMOUS;
     propFullName = propFullName || propName;
+    if (!__DEV__) {
+      if (secret !== ReactPropTypesSecret && typeof console !== 'undefined') {
+        console.error(
```

Comment:
My concern was that if we only warned once and they are calling PropType functions in multiple places they are only being warned about one of those instances. That might be annoying, especially in production, since you'd think you fixed it (since you only got one warning) and then you'd get the warning again for the next instance.